### PR TITLE
[sailfish-secrets] Make opensslcrypto plugin to link with libcrypt.

### PR DESCRIPTION
**Pull request by dcaliste**
The previous PR from @rainemak missed the plugin in crypto ;)

### CODE REVIEW

**Comment by rainemak** [comment]
Fix for openssl crypto plugin test plugin went in as part of https://github.com/sailfishos/sailfish-secrets/pull/60.
Though hide_symbols was missing from there. If you like to amend this a bit I'll land this as well. Thanks @dcaliste

**Comment by dcaliste** [comment]
Ok, for inclusion with #60, good, good. I've ammended the commit as suggested just to add the hide_symbols part.

**Comment by chriadam** [comment]
LGTM

File: rpm/sailfish-secrets.spec [hunk 1]
```diff
@@ -96,6 +96,7 @@ BuildRequires:  qt5-tools
 Summary:    Unit tests for the libsailfishcrypto library.
 Group:      System/Libraries
 BuildRequires: pkgconfig(Qt5QuickTest)
+BuildRequires: pkgconfig(libcrypto)
```
Comment (chriadam) [inline rpm/sailfish-secrets.spec]: Is this a requirement for the tests?  (is this in order to build the openssl test plugin, or?)

Comment (dcaliste) [reply rpm/sailfish-secrets.spec]: Yes, it's for openssl test plugin, the testopensslcrypto .o contains call to RAND_add symbol for instance. So it must link with libcrypto. Though RAND_add is not found in the source file. I guess it's coming from inline functions from the header?

**Pull request merged**